src/game.ts

```diff
@@ -528,13 +537,17 @@ export default class Game<P extends Player<P, B> = any, B extends Board<P, B> =
           if (submoves !== undefined) {
             possibleActions.push(action.name);
             // no sub-selections to show so just create a prompt selection of this action
-            if (skipIf === 'always' && submoves.length === 0) {
+            if (submoves.length === 0) {
               submoves = [{
                 name: action.name,
                 prompt,
                 args,
                 selections: [
-                  new Selection<P>('__action__', { prompt: action.prompt ?? playerAction.prompt, value: action.name }).resolve({})
+                  new Selection<P>('__action__', {
+                    prompt: action.prompt ?? playerAction.prompt,
+                    value: action.name,
+                    skipIf
```

Comment:
correctly preserves a skipIf=never if put on the playerActions (bug from @mvolfik)